apisix/schema_def.lua

```diff
@@ -499,6 +499,12 @@ _M.ssl = {
             type = "integer",
             minimum = 1588262400,  -- 2020/5/1 0:0:0
         },
+        status = {
```

Comment:
enable or disable ssl cert?  I think we need add more desc or a better name.

Reply:
I confused too.